pyscriptjs/Makefile

```diff
@@ -31,8 +31,12 @@ dev:
 build:
 	npm run build
 
+test-setup:
```

Comment:
I'd add this as part of the dev setup and not require the users (contributors) to run a second cmd..